components/grid/templates/popup-form-template.md

```diff
@@ -26,10 +26,13 @@ With the `FormTemplate` feature, you can customize the appearance and content of
 
 ## Specifics
 
-When using the template, the default Popup form is replaced by the declared content within the `FormTemplate` tag. This introduces the following specifics:
+When using the template, the built-in popup form is replaced by the declared content in the `FormTemplate` tag. This introduces the following specifics:
 
-* The default **Update** and **Cancel** buttons are removed. This means that the [`OnUpdate` and `OnCancel`](slug:grid-editing-overview#events) events do not fire. To modify or cancel the update of a record, you need to include custom components to manage these actions. 
-* The popup footer remains empty by design. You can [either hide it or place your custom buttons in it](slug:grid-kb-handle-empty-popup-footer).
+* The default **Update** and **Cancel** buttons are removed. This means that the [`OnUpdate` and `OnCancel`](slug:grid-editing-overview#events) events do not fire. To modify or cancel the update of a record, you need to include custom components or events to manage these actions.
+* There are [two ways to define custom Form buttons](slug:grid-kb-handle-empty-popup-footer):
+    * Use the [Form `<FormButtons>` template](slug:form-formitems-buttons).
+    * Use the [Grid `<ButtonsTemplate>`](slug:grid-templates-popup-buttons), which is empty by default when using a `<FormTemplate>`. Remove the duplicate [built-in Form Submit button with an empty `<FormButtons>` template](slug:form-formitems-buttons).
+* If you leave the Grid popup footer (`<ButtonsTemplate>`) empty, it takes up some space in the popup. You can [remove this empty popup footer space with CSS](slug:grid-kb-handle-empty-popup-footer).
```

Comment:
"Some" doesn't really bring any value right now. Consider being more specific or going with "it takes up space".